TI-reports/2024/2024-Q4-BEST-WG.md

```diff
@@ -0,0 +1,232 @@
+
+- Continue to maintain content.
+
+
+### Concise Guides
+
+#### Purpose
+
+- Artifacts that consolidate BEST practices in OSS software development and management techniques
+
+#### Current Status
+
+- TODO
+
+#### Up Next
+
+- TODO
+
+
+### Compiler Hardening Guides
+
+#### Purpose
+
+- Help C and C++ developers and those who compile C/C++ code, e.g., package maintainers, ensure that produced application binaries (libraries and executables) are equipped with security mechanisms provided by compilers against potential attacks and/or misbehavior.
+
+#### Current Status
+
+- Continued revision, updates, & enhancement, e.g., keeping the compiler options hardening guide up-to-date with upstream options additions and changes in GCC and Clang/LLVM and addressing feedback from Linux distribution communities.
+
+#### Up next
+
+- Microsoft MSVC guidance planned for 2025 (tracked in [BEST Issue 150](https://github.com/ossf/wg-best-practices-os-developers/issues/150))
+- Plan outreach activities for 2025, possibly talks aimed at C++ conferences.  
+
+
+### EDU.SIG
+
+#### Purpose
+
+- Deliver Baseline Secure Software Development Education and Certification to All.  Provide access to open and widely available education materials to all learners.
+Materials will be maximally accessible and easy to consume for all learners.
+
+#### Current Status
+
+- As noted above, LFD121 enrollment in 2024 was 9,361 and we completed all top-tier labs for it.
+- Draft course for managers developed, reviewed by LF Education, and their comments have been incorporated.
+
+#### Up Next
+
+- Complete the manager's course and have it posted.
+
+
+### Memory Safety SIG
+
+#### Purpose
+
+- The Memory Safety SIG is a group working within the OpenSSF's Best Practices Working Group formed to advance and deliver upon The OpenSSF's Mobilization Plan - Stream 4.
+
+#### Current Status
+
+- TODO
+
+#### Up Next
+
+- TODO
+
+
+### Python Hardening Guide
+
+#### Purpose
+
+- Help Python developers to create more secure code by explaining vulnerable and non-vulnerable coding patterns based on the CWE framework and rules.
+- Besides a description of each coding pattern, the guide includes executable code examples for each rule, which allow for an in-depth understanding of each pattern.
+
+#### Current Status
+
+- The group is working on adding more content for a broad range of CWE rules. The status is being tracked in issue [531](https://github.com/ossf/wg-best-practices-os-developers/issues/531).
+- Dave and Bart gave a lightning talk at SOSS Community Day EU and demoed the ongoing work at the OpenSSF booth at Open Source Summit EU 2024.
+
+#### Up Next
+
+- We want to further grow the community by raising more awareness, for instance by publishing a blog post.
+- We are inviting all interested Python coders to review the current content and/or pick a new CWE rule from [531](https://github.com/ossf/wg-best-practices-os-developers/issues/531) and contribute content.
+
+
+### Scorecard
```

Comment:
Hi @spencerschrock, @jeffmendoza, @justaugustus!
I'd like to ask you to add a Scorecard project update to this PR for tomorrow's status update of the Best Practices WG to the TAC. Feel free to update the PR as it suits you best, e.g., by leaving a comment, replacing the TODOs with a suggestion, push a commit or just ping me on Slack. Thanks a lot!